src/Compilers/CSharp/Test/CommandLine/CommandLineTestBase.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.CommandLine.UnitTests
+{
+    public abstract class CommandLineTestBase : CSharpTestBase
+    {
+        public string WorkingDirectory { get; }
+        public string SdkDirectory { get; }
+        public string MscorlibFullPath { get; }
+
+        public CommandLineTestBase()
+        {
+            WorkingDirectory = TempRoot.Root;
+            SdkDirectory = GetSdkDirectory(Temp);
+            MscorlibFullPath = Path.Combine(SdkDirectory, "mscorlib.dll");
+        }
+
+        /// <summary>
+        /// This will return a directory which contains mscorlib for use in the compiler instances created for
+        /// this set of tests
+        /// </summary>
+        /// <returns></returns>
+        protected static string GetSdkDirectory(TempRoot temp)
```

Comment:
This is the crux of the change. CoreClr by default does not have an SDK directory (the directory that contains mscorlib). In order to get these tests to pass I needed to create a fake SDK directory to use in the tests. Majority of the diff in the tests is just changing to use this directory.